package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "package.json",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

You don't have tests, so remove it and actually add the watch for tailwind, which you need in order to create the output.css file, remember?

```suggestion
    "build": "tailwindcss -o output.css",
    "watch": "tailwindcss -o output.css --watch"
```

Then you can use it once with `npm run build`, or you can leave it on **watch** mode, which will rebuild automatically for you and you can see the changes right away (recommended) with `npm run watch`.